fs/xfstests.py

```diff
@@ -65,11 +65,14 @@ def setUp(self):
                 packages.extend(['libtool'])
             else:
                 packages.extend(['libtool-bin'])
+            if '18' in self.detected_distro.version:
```

Comment:
Can we add this as an `elif` prior to this so that on every run `version` is checked only once?

Reply:
@harish-24 yes make sense will do as one info it new change in xfs test may for other distro case we also need to take care in future till it clear all other distro